lib/tasks/travis-ci.rake

```diff
@@ -0,0 +1,7 @@
+namespace :travis do
+  desc "Run all specs in except for the request specs"
+  RSpec::Core::RakeTask.new(:ci => :environment) do |task|
+    Rails.env = "test"
```

Comment:
Really? I was having trouble with `bin/rake travis:ci` but not `rake travis:ci` or `bundle exec rake travis:ci` (which is what Travis actually uses.)